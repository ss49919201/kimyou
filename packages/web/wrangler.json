{
  "$schema": "node_modules/wrangler/config-schema.json",
  "name": "kimyou",
  "main": "src/index.tsx",
  "compatibility_date": "2025-02-06"
}