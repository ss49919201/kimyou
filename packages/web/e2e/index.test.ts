import { afterAll, beforeAll, describe, expect, it } from "vitest";
import type { Unstable_DevWorker } from "wrangler";
import { unstable_dev } from "wrangler";

describe("E2E Snapshot test", () => {
  let worker: Unstable_DevWorker;

  beforeAll(async () => {
    worker = await unstable_dev("./src/index.tsx", {
      experimental: { disableExperimentalWarning: true },
    });
  });

  afterAll(async () => {
    await worker.stop();
  });

  it.each([{ route: "/" }, { route: "/homyos/generate" }])(
    "route `$route`",
    async ({ route }) => {
      const res = await worker.fetch(route);
      expect(await res.text()).toMatchSnapshot();
    }
  );
});
