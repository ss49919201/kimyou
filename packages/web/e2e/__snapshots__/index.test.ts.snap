// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`E2E Snapshot test > route \`'/'\` 1`] = `"<!DOCTYPE html><html><head><script src="https://cdn.tailwindcss.com"></script></head><body><div class="min-h-screen flex items-center justify-center"><ul class="text-2xl font-medium text-gray-900 bg-white border border-gray-200 rounded-lg dark:bg-gray-700 dark:border-gray-600 dark:text-white"><li class="w-full px-8 py-4 border-b border-gray-200 rounded-t-lg dark:border-gray-600"><a href="/montos" class="text-blue-600 hover:underline">門徒一覧</a></li><li class="w-full px-8 py-4 border-b border-gray-200 dark:border-gray-600"><a href="/homyos/generate" class="text-blue-600 hover:underline">法名生成</a></li></ul></div></body></html>"`;

exports[`E2E Snapshot test > route \`'/homyos/generate'\` 1`] = `"<!DOCTYPE html><html><head><script src="https://cdn.tailwindcss.com"></script></head><body><div class="max-w-6xl mx-auto p-6"><div class="mb-2"><a href="/" class="text-blue-600 hover:underline">← お品書きに戻る</a></div><div class="flex justify-between items-center mb-6"><h1 class="text-2xl font-bold">法名生成</h1></div><div class="flex justify-between items-center mb-6"><p class="text-l">名前を入力すると、AIが法名を生成します。</p></div><form method="get" action="/homyos/generate" class="max-w-md mb-6"><label for="default-search" class="mb-2 text-sm font-medium text-gray-900 sr-only dark:text-white">Generate</label><div class="relative flex"><input type="search" name="first-name" id="default-search" class="block w-full p-4 ps-10 text-sm text-gray-900 border border-gray-300 rounded-lg bg-gray-50 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" placeholder="名前を入力してください"/><div class="w-4"></div><button type="submit" class="w-32 text-white end-2.5 bottom-2.5 bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-4 py-2 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">生成</button></div></form></div></body></html>"`;

exports[`E2E Snapshot test > route \`'/montos'\` 1`] = `"<!DOCTYPE html><html><head><script src="https://cdn.tailwindcss.com"></script></head><body><div class="max-w-6xl mx-auto p-6"><div class="mb-2"><a href="/" class="text-blue-600 hover:underline">← お品書きに戻る</a></div><div class="flex justify-between items-center mb-6"><h1 class="text-2xl font-bold">門徒一覧</h1></div><form method="get" action="/montos" class="max-w-md mb-6"><label for="default-search" class="mb-2 text-sm font-medium text-gray-900 sr-only dark:text-white">Search</label><div class="relative flex"><input type="search" name="last-name" id="default-search" class="block w-full p-4 ps-10 text-sm text-gray-900 border border-gray-300 rounded-lg bg-gray-50 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" placeholder="門徒名を入力してください"/><div class="w-4"></div><button type="submit" class="w-32 text-white end-2.5 bottom-2.5 bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-4 py-2 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">検索</button></div></form><div class="overflow-x-auto bg-white rounded-lg shadow"><table class="w-full"><thead class="bg-gray-50"><tr><th class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">性</th><th class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">名</th><th class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">電話番号</th><th class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">性別</th><th class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">住所</th><th class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider"></th></tr></thead><tbody class="bg-white divide-y divide-gray-200"><tr class="hover:bg-gray-50"><td class="px-6 py-4">テスト性</td><td class="px-6 py-4 whitespace-nowrap">テスト名</td><td class="px-6 py-4 whitespace-nowrap">テスト電話番号</td><td class="px-6 py-4 whitespace-nowrap">男</td><td class="px-6 py-4 whitespace-nowrap">テスト住所</td><td class="px-6 py-4 whitespace-nowrap"><a href="montos/a06f0af8-bcdf-4f2d-9208-7408a90a294c" class="text-white bg-blue-700 hover:bg-blue-800 rounded-lg text-sm px-5 py-2.5 focus:outline-none}">詳細</a></td></tr></tbody></table></div></div></body></html>"`;

exports[`E2E Snapshot test > route \`'/montos/a06f0af8-bcdf-4f2d-9208-7408a…'\` 1`] = `"<!DOCTYPE html><html><head><script src="https://cdn.tailwindcss.com"></script></head><body><div class="max-w-6xl mx-auto p-6"><div class="mb-2"><a href="/montos" class="text-blue-600 hover:underline">← 門徒一覧に戻る</a></div><div class="items-center mb-6"><h1 class="text-2xl font-bold">門徒詳細</h1></div><div class="items-center p-6 rounded-lg shadow"><div class="text-xl mb-4 font-bold">性</div><div class="text-xl mb-4">テスト性</div><div class="text-xl mb-4 font-bold">名</div><div class="text-xl mb-4">テスト名</div><div class="text-xl mb-4 font-bold">電話番号</div><div class="text-xl mb-4">テスト電話番号</div><div class="text-xl mb-4 font-bold">性別</div><div class="text-xl mb-4">男</div><div class="text-xl mb-4 font-bold">住所</div><div class="text-xl mb-4">テスト住所</div><div class="text-xl mb-4 font-bold">法名</div><div class="text-xl mb-4">釋　一宗</div><div class="text-xl mb-4 font-bold">院号</div><div class="text-xl mb-4">帰命院</div><div class="text-xl mb-4 font-bold">命日</div><div class="text-xl mb-4">-</div><div class="text-xl mb-2 font-bold">次回年忌</div><div class="text-xl mb-2">-</div></div></div></body></html>"`;

exports[`E2E Snapshot test > route \`'/montos/new'\` 1`] = `"<!DOCTYPE html><html><head><script src="https://cdn.tailwindcss.com"></script></head><body><div class="max-w-6xl mx-auto p-6"><div class="mb-2"><a href="/montos" class="text-blue-600 hover:underline">← 門徒一覧に戻る</a></div><div class="items-center mb-6"><h1 class="text-2xl font-bold">門徒新規登録</h1></div><form method="post" action="/montos"><div class="items-center p-6 mb-6 rounded-lg shadow"><div class="mb-4"><lavel class="mb-2 block text-xl font-bold">性</lavel><input name="last-name" required="" class="px-2 text-gray-900 border border-gray-300 rounded-lg bg-gray-50 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"/></div><div class="mb-4"><lavel class="mb-2 block text-xl font-bold">名</lavel><input name="first-name" required="" class="px-2 text-gray-900 border border-gray-300 rounded-lg bg-gray-50 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"/></div><div class="mb-4"><p class="mb-2 block text-xl font-bold">性別</p><label class="mb-2 block text-xl font-bold"><input type="radio" name="gender" value="MAN" checked=""/> 男</label><label class="mb-2 block text-xl font-bold"><input type="radio" name="gender" value="WOMAN"/> 女</label></div><div class="mb-4"><lavel class="mb-2 block text-xl font-bold">住所</lavel><input name="address" required="" class="w-1/3 px-2 text-gray-900 border border-gray-300 rounded-lg bg-gray-50 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"/></div><div class="mb-4"><lavel class="mb-2 block text-xl font-bold">電話番号</lavel><input name="phone-number" required="" class="px-2 text-gray-900 border border-gray-300 rounded-lg bg-gray-50 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"/></div><div class="mb-4"><lavel class="mb-2 block text-xl font-bold">命日</lavel><input name="date-of-death" class="px-2 text-gray-900 border border-gray-300 rounded-lg bg-gray-50 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"/></div><div class="mb-4"><lavel class="mb-2 block text-xl font-bold">法名</lavel><input name="homyo" class="px-2 text-gray-900 border border-gray-300 rounded-lg bg-gray-50 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"/></div><div class="mb-4"><lavel class="mb-2 block text-xl font-bold">院号</lavel><input name="ingou" class="px-2 text-gray-900 border border-gray-300 rounded-lg bg-gray-50 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"/></div></div><button type="submit" class="m-auto block w-32 text-white end-2.5 bottom-2.5 bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-4 py-2 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800">送信</button></form></div></body></html>"`;
